src/App.vue

```diff
@@ -279,16 +279,15 @@ export default {
     choosingCheckOut() {
       return this.checkIn && !this.checkOut;
     },
-    firstEnabledDate() {
+    minCheckOutDate() {
       if (!this.checkIn) return null;
       return this.addDays(this.checkIn, this.getMinNights(this.checkIn));
     },
-    maximumDate() {
-      return new Date(8640000000000000);
-    },
-    nextDisabledDate() {
+    maxCheckOutDate() {
       if (!this.choosingCheckOut) return null;
-      return this.getNextDate(this.disabledDatesForCheckOut, this.addDays(this.checkIn, 1)) || this.maximumDate;
+      const nextDisabledDate = this.getNextDate(this.disabledDatesForCheckOut, this.addDays(this.checkIn, 1));
+      const maxDateValue = new Date(8640000000000000);
```

Comment:
why ?

Reply:
Because using `Infinity` causes errors when passed to some helper methods :confused:

Reply:
It was already used prior to this PR (via `this.maximumDate`)